tests/integration/cmd_push_test.go

```diff
@@ -53,6 +53,32 @@ var _ = Describe("odo push command tests", func() {
 			helper.HttpWaitFor("http://"+url, "UPDATED!", 30, 1)
 		})
 
+		It("should be able to create a file, push, delete, then push again propagating the deletions", func() {
```

Comment:
Test scenario looks good to me, however it seems only nodejs component is tested through out the test file. So can you please add some component variety in the spec, i mean add those component which are not tested in other scenarios.  For example (pick any component other than nodejs)
```
$ odo catalog list components
NAME        PROJECT       TAGS
dotnet      openshift     2.0,latest
httpd       openshift     2.4,latest
nginx       openshift     1.10,1.12,1.8,latest
nodejs      openshift     10,6,8,8-RHOAR,latest
perl        openshift     5.24,5.26,latest
php         openshift     7.0,7.1,latest
python      openshift     2.7,3.5,3.6,latest
ruby        openshift     2.3,2.4,2.5,latest
wildfly     openshift     10.0,10.1,11.0,12.0,13.0,8.1,9.0,latest
```

This way of implementation will increase the component coverage in push test for sure.

Reply:
Yeah sure! I'll do just that. Nodejs seems like the easiest since it's the smallest :)

Reply:
There is a commented test for this scenario https://github.com/openshift/odo/blob/baf308be44d1d0b648e3040e0b3d08af7921196d/tests/integration/cmd_push_test.go#L90
Maybe remove it or add to it?

Reply:
> There is a commented test for this scenario
> 
> https://github.com/openshift/odo/blob/baf308be44d1d0b648e3040e0b3d08af7921196d/tests/integration/cmd_push_test.go#L90
> 
> Maybe remove it or add to it?

@mik-dass I am in the favor of removing it because @cdrage has updated the same spec with some more improvement. Please remove the commented spec - https://github.com/openshift/odo/pull/1999/files#diff-e977fd6c6dedc4d8ed7367e136764033L88

Reply:
@cdrage Let's remove the commented spec